### PR TITLE
fix: broken start-erigon make target

### PR DESCRIPTION
**Pull request by fridrik01**
Fixes: https://github.com/berachain/beacon-kit/issues/2169

This PR fixes the `start-erigon` make target. The root cause was invalid paths used as params to docker and that erigon Dockerfile had changed to a different userid/group. To address that I specified to docker to run as user 1000:1000 as it was before it was changed (now it defaults to having a erigon user with id 3473:3473)

## Summary by CodeRabbit

- **Chores**
	- Updated configuration for running Ethereum nodes to improve user permissions and streamline execution commands.
	- Adjusted file paths for data directories and authentication secrets for better clarity and functionality.

### CODE REVIEW

**Review by chuck-bear — APPROVED** [review]
LGTM

**Comment by chuck-bear** [comment]
Tested and it's working 🎉

File: build/scripts/testing.mk [hunk 1]
```diff
@@ -200,16 +200,18 @@ start-besu: ## start an ephemeral `besu` node
...
-    -v $(PWD)/.tmp:/.tmp \
-    erigontech/erigon:latest init \
-    --datadir .tmp/erigon \
-    ${ETH_GENESIS_PATH}
+	--user 1000:1000 \
+	--rm -v $(PWD)/${TESTAPP_FILES_DIR}:/${TESTAPP_FILES_DIR} \
+	-v $(PWD)/.tmp:/.tmp \
+	erigontech/erigon:latest init \
+	--datadir /.tmp/erigon \
+	/${TESTAPP_FILES_DIR}/eth-genesis.json
```
Comment (abi87) [inline build/scripts/testing.mk:208]: nit: can this be `/${ETH_GENESIS_PATH}`

**Review by abi87 — COMMENTED** [review]
only a nit, than I can merge

**Review by nidhi-singh02 — APPROVED** [review]

File: build/scripts/testing.mk (hunk 1 above)
Comment (fridrik01) [reply build/scripts/testing.mk]: done!

**Pull request merged**